tests/testthat/_snaps/arg.md

```diff
@@ -1,88 +1,129 @@
+# matches arg
+
+    `arg` must be one of "bar" or "baz", not "foo".
+
+# gives an error with more than one arg
+
+    `arg` must be one of "bar" or "baz", not "bar".
+    i Did you mean "bar"?
+
+# gives error with different than rearranged arg vs value
+
+    `myarg` must be one of "fun" or "bar", not "foo".
+
+---
+
+    `x` must be one of "foo" or "bar", not "foo".
+    i Did you mean "foo"?
```

Comment:
This is a bit confusing. Is foo allowed or not? (I assume I'm probably reading this out of context)

Reply:
This is now:

```r
#> Error in `arg_match()`: `arg` must be length 1 or a permutation of `values`.
```

I haven't taken the time to include an enumeration of values because this is a rare error message. Normally `arg_match()` requires a length 1 vector or a length n vector in case the user hasn't supplied a value and `arg_match()` gets the argument default. In some cases, that default vector is not identical to the vector of values because it is generated in another package. A permutation of the values is then allowed. This unit test shows what happens when a length > 1 `arg` is not a permutation of `values`.